.github/workflows/all.yml

```diff
@@ -145,6 +145,8 @@ jobs:
           ./scripts/print_info.sh
 
       - name: 'Publish release'
+        # Dont try to publish for PR's
+        if: github.event_name != 'pull_request'
```

Comment:
Why? This was actually handy for testing.

Reply:
doesnt work for anyone but you I assume ill change it to just not fail on it